CHANGELOG.md

```diff
@@ -86,7 +86,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### Security
 
 ## [Unreleased 2.x]
-### Added
+### Add TokenManager Interface ([#7452](https://github.com/opensearch-project/OpenSearch/pull/7452))
```

Comment:
@scrawfor99 @peternied Can we fix up this changelog entry on main and and 2.x? The `### Added` header was replaced by this entry.

Reply:
I went ahead and included the fix in these PRs: #7945 #7946